doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -52,66 +63,48 @@ The default notification schema installed for Kaa applications is empty. For the
 }
 ```
 
-You can configure your own notification schema via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/saveNotificationSchema) call or use Administration UI as shown below. 
-
-
-The list of notification schemas created by a tenant developer for the application is shown in the **Notification schemas** window.
```

Comment:
This screenshot is given to illustrate this portion of the sentence above it: "select it from the Notification page under the Schemas section of the application".